_includes/footer.html

```diff
@@ -1,12 +1,12 @@
 <footer class="footer">
     <div class="newsletter-box">
-        <div class="newsletter-title">Stay connected</div>
-        <p>Join us today to be the first to know about our latest product launches.</p>
+        <div class="newsletter-title">Stay Connected</div>
+        <p>Join us today and be the first to know about our latest product launches.</p>
         <form id="newsletter-form" class="newsletter-form">
             <input required name="email" type="email" class="gv-input" placeholder="Email">
             <button type="submit" class="gv-button">Submit</button>
         </form>
-        <span class="message">We won’t send you spam, unsubscribe at any time</span>
+        <span class="message">We won’t send you spam (unsubscribe at any time). View our Privacy Policy <a href="https://docs.gravitee.io/apim/3.x/gravitee_privacy_policy.html">here</a>.</span>
```

Comment:
I think this page shouldn't be under `/apim/3.x`, something like `https://docs.gravitee.io/gravitee_privacy_policy.html`

Reply:
fixed! thank you for this suggestion.